src/py/flwr/server/app.py

```diff
@@ -308,15 +344,28 @@ def run_server() -> None:
 
     # Start Fleet API
     if args.fleet_api_type == "rest":
+        parsed_address = parse_address(args.rest_fleet_api_address)
+        if not parsed_address:
+            sys.exit(
+                f"Fleet IP address ({args.rest_fleet_api_address}) cannot be parsed."
+            )
+        host, port, _ = parsed_address
         fleet_thread = threading.Thread(
             target=_run_fleet_api_rest,
-            args=(args.rest_fleet_api_address, state_factory),
+            args=(host, port, state_factory),
         )
         fleet_thread.start()
         bckg_threads.append(fleet_thread)
     elif args.fleet_api_type == "grpc":
+        parsed_address = parse_address(args.grpc_fleet_api_address)
+        if not parsed_address:
+            sys.exit(
+                f"Fleet IP address ({args.grpc_fleet_api_address}) cannot be parsed."
+            )
```

Comment:
```suggestion
        address_arg = args.grpc_fleet_api_address
        parsed_address = parse_address(address_arg)
        if not parsed_address:
            sys.exit(f"Fleet IP address ({address_arg}) cannot be parsed.")
```

Reply:
@charlesbvll we can use the same approach wherever `sys.exit` get formatted to span three lines.